doc/changes/latest.inc

```diff
@@ -71,6 +71,8 @@ Enhancements
 
 - Show multiple colors and linestyles for excluded components with :class:`mne.Evoked` in :meth:`mne.preprocessing.ICA.plot_sources` (:gh:`9444` by `Martin Schulz`_)
 
+- Speed up BCD solver in :func:`mne.inverse_sparse.mixed_norm` by adding Anderson acceleration (:gh:`9481` by **new contributor** `Pierre-Antoine Bannier`_ and `Alex Gramfort`_)
```

Comment:
new contributor enhancements should be grouped toward the top of the section, and need the `|` substitution syntax (see other examples above)